### PR TITLE
Updated Russian Translation

### PR DESCRIPTION
**Pull request by TheExentist151**

### CODE REVIEW

File: HedgeModManager/Languages/ru-RU.xaml [hunk 1]
```diff
@@ -69,11 +69,19 @@
     <system:String x:Key="CodesUIInfo"    >Нажмите два раза по этому коду для большей информации.</system:String>
     <system:String x:Key="CodesUIInfoHint">Наведите курсор на код для большей информации.</system:String>
     <system:String x:Key="CodesUINoInfo"  >Нет информации.</system:String>
+	<system:String x:Key="CodesUICopyCode">Скопировать код в буфер обмена</system:String>
```
Comment (RagdollClash) [inline HedgeModManager/Languages/ru-RU.xaml]: Fix the white spacing here.

File: HedgeModManager/Languages/ru-RU.xaml [hunk 2]
```diff
@@ -69,11 +69,19 @@
...
     <system:String x:Key="CodesUIExpand"  >Раскрыть всё</system:String>
     <system:String x:Key="CodesUICollapse">Закрыть всё</system:String>
     <system:String x:Key="CodesUINullCategory">Без категории</system:String>
     <system:String xml:space="preserve" x:Key="CodesUIVersionIncompatible">Установленные коды несовместимы с загрузчиком кодов.&#x0a;Пожалуйста, обновите коды, а также загрузчик кодов.&#x0a;ВНИМАНИЕ: Это приведет к удалению всех пользовательских кодов.</system:String>
     <system:String xml:space="preserve" x:Key="CodesUIFetching">Получение кодов...</system:String>
+	
+	<!-- Diff UI Strings -->
```
Comment (RagdollClash) [inline HedgeModManager/Languages/ru-RU.xaml]: Fix the white spacing here.

**Review by RagdollClash — CHANGES_REQUESTED** [review]

File: HedgeModManager/Languages/ru-RU.xaml (hunk 1 above)
Comment (TheExentist151) [reply HedgeModManager/Languages/ru-RU.xaml]: image
What do you mean by "white space"? It looks normal to me in the editor

**Comment by TheExentist151** [comment]
Updated localization to the latest version of HMM

File: HedgeModManager/Languages/ru-RU.xaml (hunk 1 above)
Comment (RagdollClash) [reply HedgeModManager/Languages/ru-RU.xaml]: Its using a tabbed space instead of 4 spaces like the rest of them, you can see it on GitHub

**Comment by RagdollClash** [comment]
Actually just realized I can easily fix those white space errors right in GitHub.

**Review by RagdollClash — APPROVED** [review]
LGTM

**Pull request merged**